rsync-homedir-excludes.txt

```diff
@@ -93,6 +96,8 @@
 # Session-specific:
 /.dbus
 .cache
+.var/app/*/cache
```

Comment:
This

Reply:
No it isn't, is it? `Cache` is not the same as `cache` (case sensitivity).

Reply:
is there only one level between? or should it also need `**`? And does this not belong into the Electron Apps section too?

Reply:
These are the flatpak equivalent of `~/.cache`. Not necessarily a Electron app.

Reply:
so Electron and flatpak both use `.var/app` ?

Reply:
No, flatpak apps use `.var/app`. Electron apps can be installed as a normal app (from your distro's repo), or as a flatpak